packages/preview-middleware-client/types/sap.ui.rta.d.ts

```diff
@@ -240,7 +241,8 @@ declare module 'sap/ui/rta/RuntimeAuthoring' {
         } & Component;
         stop: (bSkipSave, bSkipRestart) => Promise<void>;
         attachStop: (handler: (event: Event) => void) => void;
-        setMode: (sNewMode: string) => void;
+        getMode: () => RtaMode;
```

Comment:
Is this method available for all ui5 versions or only higher ones? If it's not lower versions will throw error

Reply:
Checked that it works in lower versions including 1.71.70